docs/rst/recording/usage/configuration.rst

```diff
@@ -368,7 +368,7 @@ Enabling ``file-rotation`` allows the |ddsrecorder| to overwrite old files to fr
 
 .. note::
 
-    If the *DDS Recorder's* state changes to ``stop`` while ``file-rotation`` is set, the |ddsrecorder| will save its current output files from being overwritten in the future.
+    To keep the |ddsrecorder| from overwriting previous output files, the :ref:`Remote Controller <recorder_remote_control>` can send a ``stop`` signal with the tag ``avoid_overwriting_output`` set.
```

Comment:
I believe you should be explicit on how to activate this feature (same as it's done with next state).

Reply:
NIT: I think quotation marks would be mandatory for a string key. Also please replace "signal" for "command", to keep the same terminology, and be also explicit about the argument you are referring to ("args"). Maybe a pointer to section 4.2 where command type is defined would be helpful.